docs/en/sql-reference/functions/uuid-functions.md

```diff
@@ -18,39 +18,213 @@ generateUUIDv4([x])
 
 **Arguments**
 
-- `x` — [Expression](../../sql-reference/syntax.md#syntax-expressions) resulting in any of the [supported data types](../../sql-reference/data-types/index.md#data_types). The resulting value is discarded, but the expression itself if used for bypassing [common subexpression elimination](../../sql-reference/functions/index.md#common-subexpression-elimination) if the function is called multiple times in one query. Optional parameter.
+- `x` — [Expression](../../sql-reference/syntax.md#syntax-expressions) resulting in any of the [supported data types](../../sql-reference/data-types/index.md#data_types). The expression is used to bypass [common subexpression elimination](../../sql-reference/functions/index.md#common-subexpression-elimination) if the function is called multiple times in a query but otherwise ignored. Optional.
 
 **Returned value**
 
-The UUID type value.
+A value of type UUIDv4.
 
-**Usage example**
+**Example**
 
-This example demonstrates creating a table with the UUID type column and inserting a value into the table.
+First, create a table with a column of type UUID, then insert a generated UUIDv4 into the table.
 
 ``` sql
-CREATE TABLE t_uuid (x UUID) ENGINE=TinyLog
+CREATE TABLE tab (uuid UUID) ENGINE = Memory;
 
-INSERT INTO t_uuid SELECT generateUUIDv4()
+INSERT INTO tab SELECT generateUUIDv4();
 
-SELECT * FROM t_uuid
+SELECT * FROM tab;
 ```
 
 ```response
-┌────────────────────────────────────x─┐
+┌─────────────────────────────────uuid─┐
 │ f4bf890f-f9dc-4332-ad5c-0c18e73f28e9 │
 └──────────────────────────────────────┘
 ```
 
-**Usage example if it is needed to generate multiple values in one row**
+**Example where multiple UUIDs are generated per row**
 
 ```sql
-SELECT generateUUIDv4(1), generateUUIDv4(2)
+SELECT generateUUIDv4(1), generateUUIDv4(2);
+
 ┌─generateUUIDv4(1)────────────────────┬─generateUUIDv4(2)────────────────────┐
 │ 2d49dc6e-ddce-4cd0-afb8-790956df54c1 │ 8abf8c13-7dea-4fdf-af3e-0e18767770e6 │
 └──────────────────────────────────────┴──────────────────────────────────────┘
 ```
 
+## generateUUIDv7
```

Comment:
I asked myself what people actually expect when they mean "UUIDv7". Is it this one or the counter-based versions (my "feeling" is that it is the latter)? The version which people expect should be `generateUUIDv7`.

Reply:
Sounds good!